src/TreeTransform.jl

```diff
@@ -290,7 +286,6 @@ function xform(ctx::RewriteContext, @nospecialize(data))
         end
```

Comment:
on line 266, could that be written as `rewritten = ctx.fixed_point.(node)`? It's a little unusual to see `broadcast` called explicitly